package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "build:babel": "yarn workspaces foreach -pt run build:babel",
-    "build:typecheck": "tsc -b -v packages/example-lib",
+    "build:typecheck": "tsc -b -v packages/example-lib apps/orcid-user-profile-script",
```

Comment:
Can we leverage `yarn workspace foreach` to have each package we need to typecheck?
Not an action for now I'm just wondering how to chain this operation.

Reply:
Yeah so fortunately we only need to specify the final things we want to build in here, which would be `apps/*`, and TSC will take care of building all the dependencies from `packages/*` that they require. However, unfortunately, the 'special' CRA frontend cannot be typechecked in this same simple way. We probably just want to specify 'all packages and apps except the frontend' in here, but I guess that requires writing a small JS script.

Reply:
I will note this down and think of a solution